### PR TITLE
feat: add helmfile for Digital Ocean cluster

### PR DESCRIPTION
**Pull request by lemeurherve**
Part of https://github.com/jenkins-infra/helpdesk/issues/2651

### CODE REVIEW

File: updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml [hunk 1]
```diff
@@ -20,14 +20,22 @@ sources:
       name: jenkins-kubernetes-agents
 
 targets:
-  updateChartVersion:
+  updateCik8sChartVersion:
```
Comment (dduportal) [inline updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml:23]: Don't forget to update this target in the pull request list below

File: updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml [hunk 2]
```diff
@@ -20,21 +20,30 @@ sources:
...
       replacePattern: 'chart: jenkins-infra/jenkins-kubernetes-agents${1}version: {{ source `lastChartVersion` }}'
     scmID: default
+  updateDoksChartVersion:
+    name: "Update the chart version for jenkins kubernetes agent"
+    kind: file
+    spec:
+      file: clusters/doks.yaml
+      matchPattern: 'chart: jenkins-infra\/jenkins-kubernetes-agents((\r\n|\r|\n)(\s+))version: .*'
+      replacePattern: 'chart: jenkins-infra/jenkins-kubernetes-agents${1}version: {{ source `lastChartVersion` }}'
+    scmID: default
```
Comment (dduportal) [inline updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml:38]: Damn, the current "updatecli diff" process is blocking the whole "helmfil diff" on the build for the PR.
Do you mind if we comment out the code changed in this updatecli manifest for this PR and uncomment on na subsqeunt PR?

Comment (lemeurherve) [reply updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml:38]: WDYT about spliting out updatecli to its own pipeline?

Comment (dduportal) [reply updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml:38]: That way out of the scope of this PR: yes if you are willing to it somewhere else but let's start by finishing this PR first :)

File: Jenkinsfile_k8s [hunk 3]
```diff
@@ -74,6 +74,7 @@ pipeline {
         }
       environment {
         KUBECONFIG            = credentials("kubeconfig-${K8S_CLUSTER}")
+        CURRENT_CLUSTER       = "${K8S_CLUSTER}"
```
Comment (lemeurherve) [inline Jenkinsfile_k8s:77]:
```suggestion
```

**Review by dduportal — APPROVED** [review]
🚢 👏

**Pull request merged**